rpcs3/Emu/Cell/Modules/cellKb.cpp

```diff
@@ -33,15 +33,11 @@ error_code cellKbInit(u32 max_connect)
 {
 	sys_io.warning("cellKbInit(max_connect=%d)", max_connect);
 
-	auto handler = fxm::get<KeyboardHandlerBase>();
-
-	if (handler)
-		return CELL_KB_ERROR_ALREADY_INITIALIZED;
+	const auto handler = g_fxo->get<KeyboardHandlerBase>();
```

Comment:
Wtf is this

Reply:
Missing error checking..

Reply:
The whole commit is gonna break the modules